templates/include/amqp-kafka-bridge.jsonnet

```diff
@@ -29,12 +31,66 @@ local common = import "common.jsonnet";
           },
           "spec": {
             "containers": [
-              common.clientContainer("amqp-kafka-bridge", image_repo, "512Mi", [
-                        {
-                          "name": "KAFKA_BOOTSTRAP_SERVERS",
-                          "value": "${KAFKA_BOOTSTRAP_SERVERS}"
-                        }], true, true),
-            ]
+              {
+                "image": image_repo,
+                "name": "amqp-kafka-bridge",
+                "env": [
+                  {
+                    "name": "AMQP_CERT_DIR",
+                    "value": "/etc/enmasse-certs"
+                  },
+                  {
+                    "name": "KAFKA_BOOTSTRAP_SERVERS",
+                    "value": "${KAFKA_BOOTSTRAP_SERVERS}"
+                  }
+                ],
+                "resources": {
+                    "requests": {
+                        "memory": "512Mi"
+                    },
+                    "limits": {
+                        "memory": "512Mi"
+                    }
+                },
+                "volumeMounts": [
+                  {
+                    "name": certSecretName,
+                    "mountPath": "/etc/enmasse-certs",
+                    "readOnly": true
+                  }
+                ],
+                "ports": [
+                  {
+                    "name": "health",
+                    "containerPort": "8080"
+                  },
+                  {
+                    "name": "ready",
+                    "containerPort": "8080"
+                  }
+                ],
+                "livenessProbe": {
+                  "httpGet": {
+                    "path": "/health",
+                    "port": "health"
+                  }
+                },
+                "readinessProbe": {
+                  "httpGet": {
+                    "path": "/ready",
+                    "port": "ready"
+                  }
+                },
+              }
+            ],
+            "volumes": [
+              {
+                "name": certSecretName,
+                "secret": {
+                  "secretName": certSecretName
+                }
+              }
+             ]
```

Comment:
maybe we could try to extract a function like the `clientContainer` as before to be less verbose here ?

Reply:
IMHO, I'd say this is fine for now. We have an outstanding task to look at ksonnet and get a better structure of the jsonnet files in place soon. Although verbose, it's a bit easier to understand.

Reply:
I probably took a bit of a shortcut here :-). I did a quick search and it seems that the clientContainer function from `common.jsonnet` is not used for anything anymore. So it should be easy to add some support for volume mounts into it.

Reply:
I updated the PR so that it uses the function again. Have a look.